### PR TITLE
Add Mayhem

### PR DESCRIPTION
**Pull request by nottirb**
- Added a Dockerfile for fuzz testing
- Added Mayhemfile(s) for fuzz testing
- Added a `mayhem.yml` file for automatically running fuzz tests with Mayhem using the `mcode` action

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+project: adsb-deku
```
Comment (xansec) [inline Mayhemfile:1]: Is it possible to rename this from fuzz_target_1 to something that describes what's being fuzzed? I know this was what it was called upstream but I'd prefer if we can clarify targets

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 1 above)
Comment (nottirb) [reply Mayhemfile:1]: I'll work on getting this changed. This change makes sense to me.

**Comment by xansec** [comment]
Thanks, LGTM!

**Comment by nottirb** [comment]
It looks like there might be a slight issue here, though I'm not quite sure what it is. For some reason the mcode action is saying that it isn't able to run the command in the mayhemfile because it doesn't exist.

I pulled the image that the github action built and pushed, but when I checked it, it had the command/binary and was able to run on my computer just fine. (Building the Dockerfile and calling ``mayhem run -f mayhem/frame_from_bytes.mayhemfile .`` would work on my machine)

Any idea why this might be happening?

**Pull request merged**